### PR TITLE
Toggle button for question reminder in Inequality

### PR DESCRIPTION
**Pull request by Morpheu5**
This adds a way of hiding the question reminder in Inequality which sometimes can block out the actual editor.

Fair warning: `pointer-events` is an evil CSS property that can interfere with browser mouse events.

### CODE REVIEW

File: src/scss/common/inequality-modal.scss [hunk 1]
```diff
@@ -16,16 +16,30 @@
 
     .question-reminder {
         position: absolute;
-        bottom: 140px;
+        bottom: 30px;
         left: 250px;
         right: 250px;
-        text-align: center;
+        text-align: justify;
```
Comment (bh412) [inline src/scss/common/inequality-modal.scss:22]: This PR seems to move the text to be left aligned, maths stays central though

File: src/scss/common/inequality-modal.scss [hunk 2]
```diff
@@ -16,16 +16,30 @@
...
         border-radius: 20px;
         background-color: rgba(0, 0, 0, 0.6);
         color: white;
-        padding: 1.25rem 1px 0;
+        padding: 1.25rem;
         pointer-events: none;
         z-index: 255 !important;
+
+        .reminder-toggle {
```
Comment (bh412) [inline src/scss/common/inequality-modal.scss:30]: Slightly worried that this doesn't achieve what it's trying to address. The box is still visible on small screens

**Review by bh412 — CHANGES_REQUESTED** [review]
On mobile screens the question is even bigger now with the button within it, and when it's 'hidden' the button and full width grey box is still there. Would it be possible to move the show/hide button out to a hexagon, like the 'centre' button and fully hide the grey question reminder box when 'hidden'. 
I'm also worried about the comment on the PR, you say it sometimes hides the full equation editor?

File: src/scss/common/inequality-modal.scss (hunk 2 above)
Comment (Morpheu5) [reply src/scss/common/inequality-modal.scss:30]: Yeah, the alternative is to have an extra hexagonal button. I might do that if I can't move the grey box enough out of the way without removing it entirely.

File: src/scss/common/inequality-modal.scss (hunk 1 above)
Comment (Morpheu5) [reply src/scss/common/inequality-modal.scss:22]: The question linked in the card is horrendously long and it looks much better when justified. Math is always centred when on a line of its own.

Comment (bh412) [reply src/scss/common/inequality-modal.scss:22]: Ok - felt I should flag and make sure it's a conscious decision as it changes behaviour from current

File: src/scss/common/inequality-modal.scss (hunk 2 above)
Comment (bh412) [reply src/scss/common/inequality-modal.scss:30]: Not ideal but I'm not sure how to get around the grey box issue, having just the current button with no grey box around would also look very odd

Comment (Morpheu5) [reply src/scss/common/inequality-modal.scss:30]: I fixed this by adding an extra button.

**Review by bh412 — APPROVED** [review]
Works as intended now and hides question box

**Pull request merged**